content/p-vs-np/en/p-vs-np.md

```diff
@@ -307,7 +306,7 @@ We can substitute the expressions for $A > B$ and $A = B$ formula in to the equa
 <source src="https://video.wixstatic.com/video/935a00_655f627a117f46ecb0a48deb2640b9a1/1080p/mp4/file.mp4" type="video/mp4">
 </video>
 
-The final Boolean formula that expresses P ≥ Q is:
+The final Boolean formula that expresses P ≥ Q, for 4 bits, is:
```

Comment:
$P \geq Q$

Reply:
Sorry, do you mean I should keep the original? I just wanted to emphasize that this formula is for 4 bits, since readers tend to forget these details.

Reply:
The change is good, while we're at changing the line, let's add the math notation.

Reply:
ah heh, the comment didn't show the symbols. will fix.